pwndbg/gdblib/file.py

```diff
@@ -107,7 +107,7 @@ def readlink(path):
 
     if is_qemu:
         if not os.path.exists(path):
-            path = os.path.join(pwndbg.gdblib.qemu.root(), path)
+            path = os.path.join(pwndbg.gdblib.qemu.root() or "", path)
```

Comment:
It's not obvious to me why we need the `or ""`, it may be worth adding a comment.